src/js/electron/main.ts

```diff
@@ -47,7 +47,7 @@ async function main() {
   // autoUpdater should not run in dev, and will fail if the code has not been signed
   if (!electronIsDev) {
     try {
-      setupAutoUpdater()
+      await setupAutoUpdater()
```

Comment:
Is the await is necessary. I don't think we want to hold up this main function to wait for the autoUpdater code to resolve. The `.catch()` syntax will let the main function continue running.